sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/PublicSurfaceTests.cs

```diff
@@ -9,6 +9,7 @@ namespace Microsoft.Azure.WebJobs.Host.UnitTests
     public class PublicSurfaceTests
     {
         [Test]
+        [Ignore("new major version")]
```

Comment:
With ApiCompat in the mix, do we need this test any longer?

Reply:
Let's remove this one.